docs/userguide/execution.rst

```diff
@@ -39,7 +39,7 @@ parameters include access keys, instance type, and spot bid price
 Parsl currently supports the following providers:
 
 1. `parsl.providers.LocalProvider`: The provider allows you to run locally on your laptop or workstation.
-2. `parsl.providers.CobaltProvider`: This provider allows you to schedule resources via the Cobalt scheduler.
+2. `parsl.providers.CobaltProvider: This provider allows you to schedule resources via the Cobalt scheduler. Please note that the CobaltProvider is deprecated and will be removed by 2024.04.
```

Comment:
this looks you like were also working on a different issue, but in the same git branch? This isn't part of issue #3161, and you should try to keep pull requests focused on just one thing